app/controllers/contacts_controller.rb

```diff
@@ -3,6 +3,7 @@
 class ContactsController < ApplicationController
   before_action :set_contact, only: %i[edit update show needs add_needs]
 
+  # to do - search with policy scope?
```

Comment:
I haven't done much work into checking if the roles work with the search functionality as it was merged later after I had started, will look into this now though

Reply:
I've asked in the dev channel.  I think we would restrict roles to either be able to search or not.  We don't have the time to be more fine grained at the moment.

Reply:
The PR has been updated to make search operate on scoped contacts